server/middleware/configureEstablishment.js

```diff
@@ -24,12 +15,11 @@ const configureEstablishment = () => (req, res, next) => {
     if (typeof establishmentId !== 'undefined') {
       req.session.establishmentId = establishmentId;
       req.session.establishmentName = establishmentName;
-      req.session.establishmentPersonalisationEnabled =
-        getEstablishmentPersonalisation(establishmentId);
     }
   }
 
   res.locals.feedbackId = uuid();
+  res.locals.establishmentName = req.session.establishmentName;
   res.locals.establishmentDisplayName = `${getEstablishmentDisplayName(
     req.session.establishmentId,
   )}`;
```

Comment:
I appreciate this is out of scope but why is this a string variable? A glorified toString()?

Reply:
dunno